todo/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./reset.css" />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>ToDoList</title>
+  </head>
+  <body>
+    <header>
+      <h1 class="title">TodoList</h1>
+      <nav class="menu_bar">
+        <p class="ir">메뉴바입니다.</p>
+        <button type="button">오늘만 보기</button>
+        <button type="button">내일만 보기</button>
+        <button type="button">함께 보기</button>
+      </nav>
+    </header>
+    <main>
+      <section class="today-todos">
+        <h2>오늘 할 일</h2>
+        <ul class="todos"></ul>
+        <article class="add-form">
+          <input
+            type="text"
+            class="add-input today-input"
+            placeholder="오늘 할 일을 입력하세요."
+          />
+          <button type="button" class="add-button today-button">+</button>
+        </article>
+      </section>
+      <section class="tomorrow-todos">
+        <h2>내일 할 일</h2>
+        <ul class="todos"></ul>
+        <article class="add-form">
+          <input
+            type="text"
+            class="add-input tomorrow-input"
+            placeholder="내일 할 일을 입력하세요."
+          />
+          <button type="button" class="add-button tomorrow-button">+</button>
+        </article>
+      </section>
```

Comment:
[[[html 태그] section, article 태그 (사용, 비교)](https://abcdqbbq.tistory.com/61)를 참고해봤는데...! article은 현재 페이지에서 독립을 시킬수있는 부분인데, input과 button은 이 리스트에 할일을 추가하기 위해서 독랍할수 없자나..!!!

그래서 나는 article보다는 input과 button을 사용하고 있으니까 !! form태그로 감싸주고 h2부터 section전까지 article로 감싸주는게 맞는게 아닌가 싶어!!

```suggestion
      <section class="tomorrow-todos">
        <article class="tommorrow-todos__content">
             <h2>내일 할 일</h2>
             <ul class="todos"></ul>
             <form  class="add-form" action="http..." accept-charset="utf-8" name="todos_info" method="post" >
               <input
                type="text"
                 class="add-input tomorrow-input"
                 placeholder="내일 할 일을 입력하세요."
               />
              <button type="button" class="add-button tomorrow-button">+</button>
             </form>
        </article>
 </section>
```

Reply:
아하 맞아 form 태그 생각을 못 했네!! 적용시켜봐야겠다